scripts/mas_sso_config.sh

```diff
@@ -11,11 +11,14 @@ PASS=admin
 FLEET_OPERATOR_ROLE=kas_fleetshard_operator
 CONNECTOR_OPERATOR_ROLE=connector_fleetshard_operator
 
+# wait for keycloak container to be up or timeout after 2 minutes
+ERR_MESSAGE="Keycloak server not running at localhost:8180. No realm configuration will be applied"
+timeout 120 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost:8180/auth/realms/master)" != "200" ]]; do echo "Waiting for keycloak server at localhost:8180"; sleep 10; done' || echo $ERR_MESSAGE
```

Comment:
not sure if the `timeout` command is in some systems e.g `macOs`. Would love for someone with that OS to try it out. Maybe @akoserwal or @pawelpaszki ?

Reply:
I tried, and it works.

Reply:
Thanks @akoserwal

Reply:
@machi1990 `ubuntu` is great but macOs also do such stuff..:P